blog/2022-03-02-os.mdx

```diff
@@ -0,0 +1,13 @@
+---
+slug: os-iasql
+title: IaSQL is now open source
+authors: [depombo, dfellis, aguillenv]
+---
+
+We are excited to announce that IaSQL is now open source. The main repository is under https://github.com/iasql/iasql-engine! As perfectionists, we feel like IaSQL will never be truly ready. However, we believe IaSQL is at the point where it can start to be useful for developers managing infrastructure in the cloud. IaSQL is a SaaS that lets you model your infrastructure as data by maintaining a 2-way connection between your AWS account and a Postgres SQL database to represent the definitive state (and status) of your cloud which cannot be achived with a static infrastructure declaration. This means that when you connect an AWS account to an IaSQL instance it automatically backfills the database with your existing cloud resources. IaSQL also makes it possible to version control infrastructure changes, akin to [IaC](https://en.wikipedia.org/wiki/Infrastructure_as_code), using any migration system or idempotent stored procedures.
+
+<img src="https://iasql.com/lib_TQbMwqDvYvWhOqVJ/ae7kf13uek0k6mul.gif" alt="drawing" width="400"/>
+
+PostgreSQL IaSQL databases can be provisioned and configured via the [CLI](https://github.com/iasql/cli) which is written in Rust. The CLI calls the [engine](https://github.com/iasql/iasql-engine) which is a Node.js server written in Typescript that provisions unmodified PostgreSQL instances loaded with tables representing AWS services controlled via the [AWS SDK](https://www.npmjs.com/package/aws-sdk). AWS is our main focus at the moment, but we plan to support GCP, Azure and other cloud providers soon. This is an [updated list](https://github.com/iasql/iasql-engine#cloud-providers) of the AWS services that we currently support. Let us know if you need a specific AWS service and we should be able prioritize it!
+
+We want to make it easier for community members to write IaSQL modules, easier to use IaSQL with migration systems, add more supporting documentation, reduce waiting times when provisioning infrastructure, add more functionality to the existing AWS services and simplify some of the existing tables. The list of things we want to build into IaSQL is long, but we want to do it in the open with your feedback and help. Drop us a line on [Discord](https://discord.com/invite/machGGczea)!
```

Comment:
I think the first sentence drags on a bit too long. Can we drop it from the 6 examples to just 3 biggest ones, maybe?

Reply:
Yes